provision/rosa-cross-dc/keycloak-benchmark-crossdc-tests/src/test/java/org/keycloak/benchmark/crossdc/EntityReplicationTest.java

```diff
@@ -1,15 +1,24 @@
 package org.keycloak.benchmark.crossdc;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
 import org.keycloak.admin.client.resource.UsersResource;
+import org.keycloak.benchmark.crossdc.junit.tags.ActiveActive;
+import org.keycloak.benchmark.crossdc.junit.tags.ActivePassive;
 import org.keycloak.representations.idm.UserRepresentation;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.keycloak.benchmark.crossdc.util.KeycloakUtils.getCreatedId;
 
+@ActiveActive
+@ActivePassive
```

Comment:
Good point! What I can do is actually invert the configuration, so we exclude tags instead of include, that way we only need to annotate deployment specific tests. E.g. with the current testsuite `FailoverTest#logoutUserWithFailoverTest` is the only test that needs to be annotated with `@ActivePassive`